Articles/Blog/2021-12-HPCAndScienceProcessCulture.md

```diff
@@ -84,11 +84,11 @@ So I guess the question that I've raised in this essay is: "Is Scientific HPC A
 Carlo Graziani is a computational scientist at Argonne National Laboratory. He started out as an astrophysicist and still dabbles in the stuff, as well as in FLASH simulations, but he mostly works on machine learning and uncertainty quantification (defined as "statistics funded by the US Department of Energy"), in applications that range from weather forecasting to plasma physics to COVID-19 modeling.
 
 
-[1]: The beginnings of open-science HPC as a sustained activity&mdash;as distinct from classified science HPC conducted at US DOE national labs&mdash;can probably be dated to DOE's ASC Alliance program in the late 1990s.
```

Comment:
To be clear...what you had here were **NOT** *footnotes*. They were *invalid* GFM link definitions. The change you just made here in this commit makes them *valid* GFM link definitions.